script/lint/link-eslint-plugin.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-console */
+const fs = require('fs');
+const path = require('path');
+
+function linkEslintPlugin(runPath = process.cwd()) {
```

Comment:
Check npm link

Reply:
npm link too complicated (need package.json with specific naming, deletes files sometimes) while creating a symlink is simple.